src/test/java/io/cdap/plugin/salesforce/SalesforceSchemaUtilTest.java

```diff
@@ -132,9 +132,12 @@ public void testGetSchemaWithFields() {
       Schema.Field.of("SumRelFunc", Schema.of(Schema.Type.LONG)),
       Schema.Field.of(contacts, Schema.arrayOf(Schema.recordOf(contacts,
         Schema.Field.of("FirstName", Schema.of(Schema.Type.STRING)),
-        Schema.Field.of("Owner.Status", Schema.of(Schema.Type.STRING))))));
+        Schema.Field.of("Owner_Status", Schema.of(Schema.Type.STRING))))));
 
-    Assert.assertEquals(expectedSchema.toString(), actualSchema.toString());
+    Assert.assertEquals(expectedSchema, actualSchema);
+
+    // Parse the schema as Avro schema to make sure all the naming are accepted by Avro
```

Comment:
Nit: "...make sure all the **names** are accepted by Avro"

Reply:
fixed